### PR TITLE
JPN-482

### PR DESCRIPTION
**Pull request by perjg**
Misc formatting fixes:
- YOUR RANK and CONTRIBUTIONS are not properly aligned
- Improve indentation of long user names near avatar [~kamil]/[~akoronowicz] knows details
- On modal add padding above zero state message
- name of the user + optional admin subtitle aligned vertically with avatar (not aligned to the top) in All Contributors Module

### CODE REVIEW

File: extensions/wikia/CommunityPage/styles/components/_contributors-module.scss [hunk 1]
```diff
@@ -10,13 +11,28 @@
...
 		line-height: 21px;
 		vertical-align: middle;
 		width: calc(100% - 50px);
 	}
 
+	.community-page-top-contributor-details {
+		@include display-inline-flex();
+		line-height: 21px;
+		vertical-align: middle;
+		width: calc(100% - 50px);
```
Comment (kamilkoterba) [inline extensions/wikia/CommunityPage/styles/components/_contributors-module.scss]: Let's calculate this 50px using on avatar width var and avatar right margin

File: extensions/wikia/CommunityPage/styles/components/_contributors-module.scss [hunk 2]
```diff
@@ -10,13 +11,28 @@
...
 	.community-page-contributor-details {
 		display: inline-block;
 		line-height: 21px;
 		vertical-align: middle;
 		width: calc(100% - 50px);
 	}
 
+	.community-page-top-contributor-details {
+		@include display-inline-flex();
+		line-height: 21px;
```
Comment (kamilkoterba) [inline extensions/wikia/CommunityPage/styles/components/_contributors-module.scss]: Let's add variable for line-height as it's reused in some places.

**Comment by kamilkoterba** [comment]
LGTM

**Comment by perjg** [comment]
@kamilkoterba @adamkarminski 
Adam has been working on a separate issue which has a lot of overlap with this one:
https://github.com/Wikia/app/pull/10504

After discussion, we've decided to move forward with Adam's PR, and take in the things that are not overlapping from this PR into that one. When that is done, we should close this one without merging.

**Pull request closed without merging**